.php_cs.dist

```diff
@@ -47,7 +47,7 @@ return PhpCsFixer\Config::create()
         ],
         'phpdoc_order' => true,
         'strict_comparison' => true,
-        'strict_param' => true,
+        'strict_param' => false,
```

Comment:
I have disabled this fixer because I'm not sure as of the moment whether this will break any existing code.

Reply:
It was enable because the changed code with this rule seemed safe:

https://github.com/friendsoftwig/twigcs/blob/27541787209a57b6cbcbde46826df25b37b8457f/src/Rule/ForbiddenFunctions.php#L63

```diff
-        if (!in_array($function, $this->functions)) {
+        if (!in_array($function, $this->functions, true)) {
```

https://github.com/friendsoftwig/twigcs/blob/27541787209a57b6cbcbde46826df25b37b8457f/src/Rule/ForbiddenFunctions.php#L39

```diff
-                in_array($token->getValue(), $this->functions)
+                in_array($token->getValue(), $this->functions, true)
```

https://github.com/friendsoftwig/twigcs/blob/27541787209a57b6cbcbde46826df25b37b8457f/src/Rule/LowerCaseVariable.php#L26

```diff
-                if (in_array($tokens->look(-2)->getValue(), ['set', 'for'])) {
+                if (in_array($tokens->look(-2)->getValue(), ['set', 'for'], true)) {
```

Reply:
@franmomu 

I have reverted the change!